specify/specify7
# Fix formatted tree ranks in Queries

Fixes #3005

**_The Issue_**:
Any formatted tree Query Field in a query with base table other than the associated tree caused an Attribute Error to be thrown when the Query was executed. 

**To Test**:
In addition to testing the attached video in https://github.com/specify/specify7/issues/3005#issue-1587711142: 
- Try performing a query with formatted ranks where the base table of the query is the tree in question
  - For example, construct a query with Geography Query Fields from a Geography base table
  
- Try a query with multiple Query Fields from different trees from a base table containing more than one tree relationship
  - Such as CollectionObject (`Determination->Taxon`, `Locality-> Geography`)
  
- Add other field/relationships in tandem with the formatted tree fields and ensure the desired output
